s3parcp.go

```diff
@@ -15,11 +14,10 @@ import (
 	"github.com/chanzuckerberg/s3parcp/s3utils"
 )
 
-// Update this with new versions
-const version = "0.2.1-alpha"
+// to be set with `-ldflags "-X main.version="`
```

Comment:
fyi we have code in go-misc for this https://github.com/chanzuckerberg/go-misc/blob/master/ver/version.go

Reply:
Thanks, goreleaser sets this flag automatically so I think it is simple to leave it in this form for now